src/components/v5/payments/TmpAdvancedPayments.tsx

```diff
@@ -66,6 +71,7 @@ const TmpAdvancedPayments = () => {
     createdInDomain: rootDomain,
     fundFromDomainId: 1,
     networkInverseFee: networkInverseFee ?? '0',
+    tokenDecimals: tokenDecimalAmount,
```

Comment:
This will only ever work if you have all payouts in one token. As we discussed, there can multiple payouts in multiple tokens (potentially with different decimals) so the decimals need to be a property of a payout, not payload.